plugins/inputs/opcua/sample.conf

```diff
@@ -6,12 +6,22 @@
   ## OPC UA Endpoint URL
   # endpoint = "opc.tcp://localhost:4840"
   #
+  ## The service set to use to retrieve/receive the values from the OPC UA server
+  # connection_service = "request"
+  # connection_service = "subscribe"
```

Comment:
Only give the default value as example. The possible values can be given in the comment.